### PR TITLE
17: Addressing some PR changes

### PR DESCRIPTION
**Pull request by davidabrahams**
I went in and addressed some of the changes I requested. Also `npm run build:test` was broken, I fixed it

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 var path = require('path');
+var glob = require("glob");
```
Comment (billmwong) [inline webpack.config.js]: what is glob

File: src/index.js [hunk 2]
```diff
@@ -9,7 +9,7 @@
 // @author        Cflakes, snaps_, altodyte, shanek21, davidabrahams, billmwong
 // @namespace     http://www.reddit.com/user/snaps_
 // @require       https://raw.githubusercontent.com/bgrins/javascript-astar/master/astar.js
-// @license       2015
+// @license       2008
```
Comment (billmwong) [inline src/index.js]: wat is this

File: .gitignore [hunk 3]
```diff
@@ -5,5 +5,6 @@
 
 node_modules/
 public/bundle.js
+public/test.js
```
Comment (billmwong) [inline .gitignore]: why does it make this, what is in this file

File: src/index.js (hunk 2 above)
Comment (davidabrahams) [inline src/index.js]: a joke haha

File: .gitignore (hunk 3 above)
Comment (davidabrahams) [inline .gitignore]: when you run `npm run build:test`, webpack both runs the tests and compiles all the code. A side effect of it running the tests is it compiles them too. I couldn't figure out how to make it not compile tests.

This is an issue that exists on master, so until we figure out how to make it not compile test.js, i figured a good thing to do was gitignore it so it stops showing up in `git status`

File: webpack.config.js (hunk 1 above)
Comment (davidabrahams) [inline webpack.config.js]: allows you to do wildcard matching. when we add more test files, they will automatically get run. (see line 12)

File: src/index.js (hunk 2 above)
Comment (shanek21) [inline src/index.js]: -_-

**Pull request merged**